styles.css

```diff
@@ -1,5 +1,12 @@
+@import url("./src/artsyUI.css");
 body{
     background: #dedede;
     font-family:  sans-serif;
     font-size: 1.6rem;
+    
+}
+.main-page-buttons button {
+    padding: 1rem;
+    border:none;
```

Comment:
Because you gave border: none, there is no button animation, if you don't want border you can give some custom animation.

Reply:
Thank u for the review. Sure i will add custom effects in due time